doc/psidk/src/development/services/standards.md

```diff
@@ -2,7 +2,7 @@
 
 ## Standard account names
 
-Trusted system services have account names which end with `-sys`. Only chain operators may create accounts with this suffix.
+Subjective system services have account names which begin with `x-`. Only the infrastructure providers may create accounts with this suffix.
```

Comment:
...How about:

## Reserved account names

By convention, apps that are specific to an individual node operator are hosted at subdomains that begin with "x-". Therefore, to avoid being overshadowed by these apps, user account names may not begin with "x-".

Reply:
fixed